src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -356,8 +360,43 @@ public Result importVoDFolder(String vodFolderPath) {
 
 		return result;
 	}
+
+	private void sendUnexpectedShutdownHook() {
```

Comment:
I think these hooks should be in the server side. Not on the application side

Reply:
i agree.  do you have a class on your mind to send this hook?